workshops/book-dash/book-dash-mcr-report.md

```diff
@@ -0,0 +1,162 @@
+  * Sketched out in notes, lots of post its! Collaboratively written across all 4 team members.
+  * Looking for reviewers.
+  
+* Tarek Allam Jr [@tallamjr](https://github.com/tallamjr):  
+  * Updating Github templates ([Issue #476](https://github.com/alan-turing-institute/the-turing-way/issues/476), [Pull request #488](https://github.com/alan-turing-institute/the-turing-way/pull/488)) [merged]
+  * Updating 'about' key as this required by Github ([Pull request #502](https://github.com/alan-turing-institute/the-turing-way/pull/502)) [merged]
+
+* Tania Allard [@trallard](https://github.com/trallard):
+  * Continuous integration for the website to test for alt tags on the images (for better accessibility) and making sure that the links resolve ([Issue #486](https://github.com/alan-turing-institute/the-turing-way/issues/486), [Pull request #513](https://github.com/alan-turing-institute/the-turing-way/pull/513)) [closed, merged]
+
+* Patricia Herterich [@pherterich](https://github.com/pherterich), Rachael Ainsworth [@rainsworth](https://github.com/rainsworth/):
+  * Reviewed pull requests. 
+  * Had a fright when Netlify got overwhelmed with all our different PRs. 
+  * Added contributors.
+  * Opened some new issues so the team have them written down for future work :smile: 
+
+* Matt:
+  * Drew all these pictures!
+  * Favourite was the community garden - nurturing everyone so that all the collaborative work can grow :heart_eyes:
+  ![https://pbs.twimg.com/media/D6zGuPiW4AI3m8s?format=jpg&name=4096x4096](https://pbs.twimg.com/media/D6zGuPiW4AI3m8s?format=jpg&name=4096x4096)
+
+
+## What did we learn?
+
+There's also an issue for feedback - any comments on what we can do better at future events are appreciated: https://github.com/alan-turing-institute/the-turing-way/issues/505
+
+* 2 first pull requests!! :bell: :bell:
+* When adding participants as collaborators to the project on GitHub, let them know and make sure they are aware of the email they've received and added functionality that comes with it.
+* Add explicit guidance on putting GitHub handles into the intro presentation and trying to make sure this happens in the initial brain storming session to keep track of contributors.
+* Feedback from [Pluses and deltas](https://hackmd.io/9IIQpagHQoOGlwUnY98xQA)
+  * Pluses: really supportive and inclusive environment, good organisation, brilliant icebreaker and really enjoyed having the artist in residence!
+  * Deltas: needed more time, friendlier introduction to Git/GitHub and more guidance on what tasks to take up.
+* Beth Montague-Hellen [@AlfAWolf140](https://github.com/AlfAWolf140) wrote a blog post about her experience: https://montaguehellen.wordpress.com/2019/05/20/the-turing-way-and-a-return-to-github/
```

Comment:
This is amazing and should be in a separate section from the feedback part I think.

Can we also capture (screen shot?) some of the tweets? There were some really great reflections that I don't want to lose.